python/rascaline/tests/utils/atomic_density.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from rascaline.utils import GaussianDensity, LodeDensity
+
+
+pytest.importorskip("scipy")
+
+
+@pytest.mark.parametrize(
+    "density",
+    [
+        GaussianDensity(atomic_gaussian_width=1.2),
+        LodeDensity(atomic_gaussian_width=1.2, potential_exponent=1),
+    ],
+)
+def test_derivative(density):
+    # Exclude very small values due instabilitues of LodeDensity
+    positions = np.linspace(1e-7, 5, num=1000)
+    dens = density.compute(positions)
+    grad_ana = density.compute_derivative(positions)
+    grad_numerical = np.gradient(dens, positions)
+
+    assert_allclose(grad_numerical, grad_ana, atol=1e-3)
```

Comment:
this tolerance is pretty high =/. I tend to go for atol~1e-9, rtol~1e-6 ideally. Here if all the values are below 1e-3, the test will do nothing.

Reply:
Sure, I can increase it!